lib/features/login/data/datasource_impl/authentication_oidc_datasource_impl.dart

```diff
@@ -25,8 +25,7 @@ class AuthenticationOIDCDataSourceImpl extends AuthenticationOIDCDataSource {
   @override
   Future<OIDCResponse> checkOIDCIsAvailable(OIDCRequest oidcRequest) {
     return Future.sync(() async {
```

Comment:
Just curious, Why do you have to put async function in `Future.sync()`, i really don't know why @@

Reply:
Because we have asynchronous tasks to be handled internally. We need it to complete before returning the value.